src/app/home/home.component.ts

```diff
@@ -116,6 +116,8 @@ export class HomeComponent implements OnInit {
     currentZoom: number;
     previousZoom: number;
 
+    public testVar: Boolean;
```

Comment:
What is this variable doing? I'm wondering if we can give it a more descriptive name.

Reply:
Oops. I thought I deleted that! I'll do that now